other/restrict_controlplane_scheduling/test.yaml

```diff
@@ -0,0 +1,18 @@
+name: restrict-controlplane-scheduling
+policies:
+  -  restrict_controlplane_scheduling.yaml
+resources:
+  -  resource.yaml
+results:
+  - policy: restrict-controlplane-scheduling
+    rule: restrict-controlplane-scheduling-master
```

Comment:
Can you also add a negative test for restrict-controlplane-scheduling-control-plane as well?

Reply:
Ok. I will do so..